.gitignore

```diff
@@ -1,3 +1,6 @@
 /target
 **/*.rs.bk
 Cargo.lock
+
+# IDEA
```

Comment:
Not sure we want to add these for each editor, may add this to a global gitignore on your machine?